tests/sentry/api/endpoints/snapshots/ProjectFiltersTest/test_get.pysnap

```diff
@@ -14,6 +14,12 @@ source: tests/sentry/api/endpoints/test_project_filters.py
   hello: localhost - Filter out events coming from localhost
   id: localhost
   name: Filter out events coming from localhost
+- active: false
```

Comment:
@andriisoldatenko @RaduW I used the following command to update this file 😉

`SENTRY_SNAPSHOTS_WRITEBACK=1 pytest tests/sentry/api/endpoints/test_project_filters.py`